modules/update-upgrading-cli.adoc

```diff
@@ -75,19 +61,21 @@ VERSION IMAGE
 4.10.12 quay.io/openshift-release-dev/ocp-release@sha256:f77f4f75c1e1a4ddd0a0355f298a834db3473fd9ca473235013e9419d1df16db
 4.10.13 quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
 ----
+[NOTE]
+*For details and information on how to perform an `EUS-to-EUS` channel upgrade, please refer to the link:https://docs.openshift.com/container-platform/4.12/updating/preparing-eus-eus-upgrade.html[Preparing to perform an EUS-to-EUS upgrade] page.
```

Comment:
is there a way for us to link here without  pinning 4.12?

Reply:
I'll look into this :)

Reply:
@wking @LalatenduMohanty  
After looking into this, I actually believe it's important to keep this up-to-date, as it still includes information for all of our still-supported versions and is maintained within our team. After reading through it, I don't see any reasons that the pining of 4.12 documentation should confuse our 4.9 customers. A couple of points stick out to me in the 4.12 version. For example:

- "You must be running at least OpenShift Container Platform 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update."

- "Support for RHEL7 workers was removed in OpenShift Container Platform 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers."

- "Node components are not updated to OpenShift Container Platform 4.9. Do not expect all features and bugs fixed in OpenShift Container Platform 4.9 to be made available until you complete the update to OpenShift Container Platform 4.10 and enable all MachineConfigPools to update."

Are there any specific concerns we should address with 4.12?

Reply:
@wking yes, kind of, though it's a little ugly and clunky. 

What @obrown1205 could do is not link the page directly in this note, since it's in a module and can't use a version agnostic xref, but instead just refer to the page title in italics here. 

Then, in the [Additional Resources section right underneath this procedure](https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli), where there's already a link to the Understanding Upgrade Channels page, she could add another xref to the EUS to EUS page (the additional resources section is located in the assembly for the entire page, not a module, so xrefs are allowed)